src/app/models/building_color_bucket_calculator.js

```diff
@@ -135,7 +135,7 @@ define(['underscore', 'd3'], function (_, d3) {
 
     var c = ["#1f5dbe", "#c4b957", "#e9a646", "#c04f31"]
     var r =  [24.8,29.1,36.0];
-    var s = d3.scale.threshold().domain(r).range(c);
+    var s = d3.scaleThreshold().domain(r).range(c);
```

Comment:
Just noting that you'll see a few instances like this as you did in the first feature branch PR that was merged. This is just part of updating D3 from V3 to V4.